peps/pep-0757.rst

```diff
@@ -246,28 +257,26 @@ Code::
     static void
     mpz_set_PyLong(mpz_t z, PyObject *obj)
     {
-        const PyLongLayout* layout = PyLong_GetNativeLayout();
         static PyLongExport long_export;
 
         PyLong_Export(obj, &long_export);
         if (long_export.digits) {
-            mpz_import(z, long_export.ndigits, layout->digits_order,
-                       layout->digit_size, layout->endianness,
-                       layout->digit_size*8 - layout->bits_per_digit,
-                       long_export.digits);
+            mpz_import(z, long_export.ndigits, int_digits_order, int_digit_size,
+                       0, int_nails, long_export.digits);
```

Comment:
I would prefer to keep endianness as a parameter, and not hardcode it.

Reply:
That hardly make sense, but I'll restore it.